icepyx/core/query.py

```diff
@@ -1,6 +1,5 @@
 import geopandas as gpd
 import matplotlib.pyplot as plt
-from pathlib import Path  # used in docstring tests
```

Comment:
Is there a best practice for cases like this where it means you now have to have the import line twice (i.e. how many times do you need to import it for examples before it's enough to import it generally?)

Reply:
None that I know of. My personal opinion is that this is a case where WET is a better practice than DRY. The examples in the tests end up in the docs, so IMO they should be self-contained and complete!